CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.2] - 2018-10-29
+### Changed
+- Se actualiza sdk js a versión 1.5.8
```

Comment:
Someone reading this changelog won't really know what changed nor why should he upgrade. Please phrase the changelog in a way that the user of plugins can understand (and note that plugins are used by people who are not developers)